backend/src/main/java/com/stock/admin/service/ExportService.java

```diff
@@ -0,0 +1,273 @@
+			docWriter = PdfWriter.getInstance(doc, new FileOutputStream(path));
+			Shop shop = shopsRepository.findByShopCode(shopCode);
+			if(shop!=null) {
+				docWriter.setPageEvent(new HeaderFooterPageEvent(shop));
+				updateDocumentHeader(doc);
+
+				// open document
+				doc.open();
+				doc.add(new Paragraph("\n\n\n\n\n"));
+				PdfPTable stocksTable = new PdfPTable(7);
+				PdfPTable infoTable = new PdfPTable(2);
+
+				updateStocksTable(stocksTable, shop, fromDate, toDate);
+				updateInfoTable(infoTable, fromDate, toDate);
+				doc.add(infoTable);
+				doc.add(new Paragraph("\n\n"));
+				doc.add(stocksTable);
+			}
+			else {
+				throw new StockAdminApplicationException(INVALID_SHOP_CODE,HttpStatus.BAD_REQUEST);
+			}
+			
+		} catch (StockAdminApplicationException stockException) {			
+			throw stockException;
+		} catch (DocumentException | FileNotFoundException e) {		
+			throw new StockAdminApplicationException(SOMETHING_WENT_WRONG, HttpStatus.INTERNAL_SERVER_ERROR);
+		} finally {
+			doc.close();
+			if (docWriter != null) {
+				// close the writer
+				docWriter.close();
+			}
+		}
+	}
+
+	/**
+	 * Update info table.
+	 *
+	 * @param infoTable the info table
+	 * @param fromDate the from date
+	 * @param toDate the to date
+	 * @throws DocumentException the document exception
+	 */
+	private void updateInfoTable(PdfPTable infoTable, Date fromDate, Date toDate) {
+
+		PdfPCell exportedDateLabel = new PdfPCell(new Phrase("Exported Date :- ", bfBold12));
+		exportedDateLabel.setBorder(Rectangle.NO_BORDER);
+		exportedDateLabel.setHorizontalAlignment(Element.ALIGN_RIGHT);
+		infoTable.addCell(exportedDateLabel);
+		PdfPCell exportedDate = new PdfPCell(new Phrase(getDate(new Date(),"IST"), bfBold12));
+		exportedDate.setBorder(Rectangle.NO_BORDER);
+		exportedDate.setHorizontalAlignment(Element.ALIGN_LEFT);
+		infoTable.addCell(exportedDate);
+		PdfPCell fromDateLabel = new PdfPCell(new Phrase("From Date :- ", bfBold12));
+		fromDateLabel.setBorder(Rectangle.NO_BORDER);
+		fromDateLabel.setHorizontalAlignment(Element.ALIGN_RIGHT);
+		infoTable.addCell(fromDateLabel);
+		PdfPCell fromDateValue = new PdfPCell(new Phrase(getDate(fromDate,"UTC"), bfBold12));
+		fromDateValue.setBorder(Rectangle.NO_BORDER);
+		fromDateValue.setHorizontalAlignment(Element.ALIGN_LEFT);
+		infoTable.addCell(fromDateValue);
+		PdfPCell toDateLabel = new PdfPCell(new Phrase("To Date :- ", bfBold12));
+		toDateLabel.setBorder(Rectangle.NO_BORDER);
+		toDateLabel.setHorizontalAlignment(Element.ALIGN_RIGHT);
+		infoTable.addCell(toDateLabel);
+		PdfPCell toDateValue = new PdfPCell(new Phrase(getDate(toDate,"UTC"), bfBold12));
+		toDateValue.setBorder(Rectangle.NO_BORDER);
+		toDateValue.setHorizontalAlignment(Element.ALIGN_LEFT);
+		infoTable.addCell(toDateValue);
+
+	}
+
+	/**
+	 * Update document header.
+	 *
+	 * @param doc the doc
+	 */
+	private void updateDocumentHeader(Document doc) {
+		// document header attributes
+		doc.addAuthor("Stock Admin");
```

Comment:
Let these string come from constant file, so that these can be changed later easily.